pyi.py

```diff
@@ -115,11 +115,16 @@ def handleNodeDelete(self, node):
 
 class PyiAwareFileChecker(checker.FileChecker):
     def run_check(self, plugin, **kwargs):
-        if self.filename.endswith('.pyi') and plugin['plugin'] == FlakesChecker:
+        if self.filename == '-':
+            filename = self.options.stdin_display_name
+        else:
+            filename = self.filename
+
+        if filename.endswith('.pyi') and plugin['plugin'] == FlakesChecker:
             LOG.info(
                 'Replacing FlakesChecker with PyiAwareFlakesChecker while '
                 'checking %r',
-                self.filename,
+                filename
```

Comment:
nit: why did you remove the trailing comma?

Reply:
Habit from JavaScript. I'll add it back.